scripts/quests/otherAreas/Elderly_Pursuits.lua

```diff
@@ -0,0 +1,183 @@
+            {
+                [518] = function(player, csid, option, npc)
+                    player:delKeyItem(xi.ki.CATHEDRAL_MEDALLION)
+                    quest:complete(player)
+                end,
+            },
+        },
+
+        [xi.zone.SOUTHERN_SAN_DORIA] =
+        {
+            ['Rouva'] =
+            {
+                onTrigger = function(player, npc)
+                    if
+                        quest:getVar(player, 'Prog') == 1 and
+                        player:hasKeyItem(xi.ki.ANTIQUE_AMULET)
+                    then
+                        return quest:progressEvent(747)
+                    elseif
+                        quest:getVar(player, 'Prog') == 3 and
+                        player:hasKeyItem(xi.ki.CATHEDRAL_MEDALLION)
+                    then
+                        return quest:progressEvent(748)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [747] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 2)
+                end,
+
+                [748] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 4)
+                end,
+            },
+        },
+
+        [xi.zone.CARPENTERS_LANDING] =
+        {
+            ['qm_para'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 2 then
+                        if quest:getVar(player, 'ParaKilled') == 1 then
+                            player:messageSpecial(carpentersLandingID.text.POLISH, xi.ki.ANTIQUE_AMULET)
+                            player:delKeyItem(xi.ki.ANTIQUE_AMULET)
+                            quest:setVar(player, 'Prog', 3)
+                            quest:setVar(player, 'ParaKilled', 0)
+                            return quest:keyItem(xi.ki.CATHEDRAL_MEDALLION)
+                        elseif
+                            GetNPCByID(carpentersLandingID.npc.qm_para):getLocalVar('engaged') < os.time() and
+                            GetNPCByID(carpentersLandingID.npc.qm_para):getLocalVar('engaged') ~= 1 and
+                            npcUtil.popFromQM(player, npc, { carpentersLandingID.mob.PARA_OFFSET }, { claim = true, hide = 0 })
+                        then
+                            GetNPCByID(carpentersLandingID.npc.qm_para):setLocalVar('engaged', 1)
+                            GetNPCByID(carpentersLandingID.npc.qm_para):setLocalVar('nextMob', carpentersLandingID.mob.PARA_OFFSET + 1)
+                            return quest:messageSpecial(carpentersLandingID.text.STENCH_OF_DECAY)
+                        end
+                    end
+                end,
+            },
+
+            ['Para'] =
+            {
+                onMobDeath = function(mob, player, optParams)
+                    if AllParaDead() then
+                        quest:setVar(player, 'ParaKilled', 1)
```

Comment:
Two questions: Do the kills still count if the player zones, if they do, why not increment the Prog value instead of introducing another variable?

Reply:
Zoning after killing Para resets the kill progress: the capture shows the player killing Para -> Zoning -> checking qm = spawn new Para.

Reply:
In that case, 'paraKilled' should be using a local var instead of one that is persisted in the db.  `quest:setLocalVar(player, ...)`